src/prolong_restrict/prolong_restrict.cpp

```diff
@@ -39,6 +40,7 @@ void Restrict(const StateDescriptor *resolved_packages,
   const auto &ref_func_map = resolved_packages->RefinementFncsToIDs();
   for (const auto &[func, idx] : ref_func_map) {
     auto restrictor = func.restrictor;
+    PARTHENON_DEBUG_REQUIRE_THROWS(restrictor != nullptr, "Valid restriction op");
```

Comment:
This should never trigger. Since it's not in setup I put it behind a `DEBUG` flag.